syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -23,9 +23,12 @@
 
 
 def _get_maximum_precision():
-    """This function returns the maximum value allowed for precision fractions before the chain decides to use LPT.
+    """
+    This function returns the maximum value allowed for precision
+    fractions before the chain decides to use LPT.
 
-    This function can be overridden if the setup requires the use of LargePrecisionTensor from a smaller precision.
+    This function can be overridden if the setup requires the use
+    of LargePrecisionTensor from a smaller precision.
```

Comment:
LargePrecisionTensor has been removed!
Maybe it's a good time to remove methods that are not used anymore :)

Reply:
@LaRiffle - So both `_get_maximum_precision()` and `default_pytorch_maximum_precision()` ?  

Also, there's a call to `_get_maximum_precision` in  the function `fix_prec()`, should I remove that too?

Reply:
Yep! :)